test/honeylemon/HoneylemonService.test.js

```diff
@@ -375,18 +444,28 @@ describe('HoneylemonService', () => {
     // await time.increase(10); // increase by 10 seconds to signify 1 day
     await fill0xOrderForAddresses(3, takerAddress, makerAddress);
 
-    await createNewMarketProtocolContract(0, mriInput, 'MRI-BTC-28D-20200502');
+    // fast forward 28 days
+    await time.increase(28 * 24 * 60 * 60 + 1);
+    // we need to deploy 28 times in order to be able to settle
+    await Promise.all(Array.from({ length: 28 }, (x, i) => {
+      return createNewMarketProtocolContract(mriInput * 28, mriInput, 'MRI-BTC-28D-20200502');
+    }));
 
     await fill0xOrderForAddresses(2, takerAddress, makerAddress);
 
     // Get contracts object from HoneyLemonService
-    const { longContracts, shortContracts } = await honeylemonService.getContracts(
+    const { longPositions, shortPositions } = await honeylemonService.getPositions(
       takerAddress
     );
 
-    const { longContracts2, shortContracts2 } = await honeylemonService.getContracts(
+    const { longPositions: longPositions2, shortPositions: shortPositions2 } = await honeylemonService.getPositions(
```

Comment:
I know this is still WIP but a test for the data returned here would be nice.

Reply:
added